src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.tsx

```diff
@@ -107,8 +110,18 @@ const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> =
             >
               {queryWrapperChildProps => {
                 const isFaceted = queryWrapperChildProps.isFacetsAvailable
+                const showFacetFilter = queryWrapperChildProps.topLevelControlsState?.showFacetFilter
                 return (
                   <>
+                    <div
```

Comment:
When the error was shown (on initial execution of the query), it may be really difficult to find since it was below the long list of facets on the left!  This moves the error banner to directly below the top level controls (which is done via css grid, but I also moved it in the DOM to make it easier to find!). Also needs to respect the facet filter collapse state

Reply:
So now it should also be obvious if you try to submit malformed SQL. Great!

Reply:
Exactly